src/doc/rustdoc/src/command-line-arguments.md

```diff
@@ -100,7 +100,16 @@ mod private { // this item is private and will not be documented
 }
 ```
 
-`--document-private-items` documents all items, even if they're not public.
+`--document-private-items` causes all items to be documented as if they were `pub`.
+
+## `--document-hidden-items`: Show items that are `#[doc(hidden)]`
+
+By default, `rustdoc` does not document items that are annotated with
```

Comment:
That's a bit weird to specify it, it's literally the original meaning. I suppose you wanted to underline that `#[doc(hidden)]` items were documented with this option? If so please reword it.

Reply:
I was mirroring the wording of `--document-private-items`